source/includes/connection-snippets/scram-snippets/java-connection.java

```diff
@@ -0,0 +1,7 @@
+// begin scram connection
```

Comment:
Let's be consistent about two things

1. Whether a full program is included.  This is a snippet, but the X509 example is a complete program.  Looking at examples for other languages, seems like complete programs are favored.
2. (Nit) Whether there is a separate local variable for the URI string.  Here there is not, but in the X509 example there is.  Looking at examples from other languages, seems like _not_ having a separate variable for the connection string is favored.

Reply:
Hey Jeff, 

to address your two points:
* SCRAM Connection snippets use a snippet since they were taken pretty much verbatim from the driver docs (See:  https://docs.mongodb.com/ecosystem/drivers/java/#connect-to-mongodb-atlas). On the other hand, X509 snippets are full runnable programs. I'm open to adjusting the SCRAM snippets to be full runnable programs but that would mean SCRAM snippets aren't consistent with the driver docs for connecting to atlas (which might be a necessary evil anyway). I would prefer to keep x509 code as full runnable code. 
* You might be referring to an old version of this PR, but now the code snippets use <cluster-url>. Let me know if this is satisfactory